cpp/ql/lib/semmle/code/cpp/PrintAST.qll

```diff
@@ -9,12 +9,12 @@
 import cpp
 private import semmle.code.cpp.Print
 
-private newtype TPrintASTConfiguration = MkPrintASTConfiguration()
+private newtype TPrintASTConfiguration = MkPrintAstConfiguration()
 
 /**
  * The query can extend this class to control which functions are printed.
  */
-class PrintASTConfiguration extends TPrintASTConfiguration {
+class PrintAstConfiguration extends TPrintASTConfiguration {
```

Comment:
Shouldn't `TPrintASTConfiguration` also be changed?

Reply:
Yes, I didn't flag those names in the QL-for-QL query. 
Fixed with https://github.com/github/codeql/pull/8323/commits/32b1d98d6e2680b031fd80221d98d0ec7ef4951e